### PR TITLE
Various improvements in Contribute section of Russian translation

### PR DESCRIPTION
**Pull request by lex111**

### CODE REVIEW

File: content/ru/docs/contribute/start.md [hunk 1]
```diff
@@ -25,12 +25,12 @@ card:
...
-[advanced](/docs/contribute/advanced/) docs contribution guides.
+Не все задачи могут быть выполнены с помощью интерфейса GitHub, но некоторые из них обсуждаются в руководствах для [продвинутых](/docs/contribute/intermediate/) и
+[опытных](/docs/contribute/advanced/) участников.
 
 ### Участие в документации SIG
 
-Документация Kubernetes поддерживается {{< glossary_tooltip text="специальной группой" term_id="sig" >}} (Special Interest Group, SIG) под названием SIG Docs. Мы [общаемся](#participate-in-sig-docs-discussions) с помощью канала Slack, списка рассылки и еженедельных видеозвонков. Будем рады новым участникам. Для получения дополнительной информации обратитесь к странице [Participating in SIG Docs](/docs/contribute/participating/).
+Документация Kubernetes поддерживается {{< glossary_tooltip text="специальной группой" term_id="sig" >}} (Special Interest Group, SIG) под названием SIG Docs. Мы [общаемся](#participate-in-sig-docs-discussions) с помощью канала Slack, списка рассылки и еженедельных видеозвонков. Будем рады новым участникам. Для получения дополнительной информации обратитесь к странице [Участие в SIG Docs](/docs/contribute/participating/).
```
Comment (sftim) [inline content/ru/docs/contribute/start.md]:
```suggestion
Документация Kubernetes поддерживается {{< glossary_tooltip text="специальной группой" term_id="sig" >}} (Special Interest Group, SIG) под названием SIG Docs. Мы [общаемся](#participate-in-sig-docs-discussions) с помощью канала Slack, списка рассылки и еженедельных видеозвонков. Будем рады новым участникам. Для получения дополнительной информации обратитесь к странице [Участие в SIG Docs](/ru/docs/contribute/participating/).
```
?

File: content/ru/docs/contribute/start.md [hunk 2]
```diff
@@ -25,12 +25,12 @@ card:
 
 Вы можете создавать новые задачи, редактировать содержимое и проверять изменения от других участников, — всё это доступно с сайта GitHub. Вы также можете использовать встроенный в GitHub поиск и историю коммитов.
 
-Не все задачи могут быть выполнены на GitHub, поэтому они обсуждаются в [intermediate](/docs/contribute/intermediate/) and
-[advanced](/docs/contribute/advanced/) docs contribution guides.
+Не все задачи могут быть выполнены с помощью интерфейса GitHub, но некоторые из них обсуждаются в руководствах для [продвинутых](/docs/contribute/intermediate/) и
+[опытных](/docs/contribute/advanced/) участников.
```
Comment (sftim) [inline content/ru/docs/contribute/start.md]:
```suggestion
Не все задачи могут быть выполнены с помощью интерфейса GitHub, но некоторые из них обсуждаются в руководствах для [продвинутых](/ru/docs/contribute/intermediate/) и
[опытных](/ru/docs/contribute/advanced/) участников.
```

File: content/ru/docs/contribute/participating.md [hunk 3]
```diff
@@ -150,7 +150,7 @@ SIG Docs активно принимает правки и дополнения
 
 - Перед одобрением пулреквеста перейдите на предварительный просмотр сайта на Netlify для сделанных изменений в PR, и убедитесь, что всё содержимое выглядит хорошо.
 
-- Участвуйте в [графике дежурства смотрителя PR](https://github.com/kubernetes/website/wiki/PR-Wranglers), чтобы вас назначили дежурным проверяющим на неделю. SIG Docs ожидает, что все утверждающие примут участие в этом графике. За подробностям обратитесь к странице [Be the PR Wrangler for a week](/docs/contribute/advanced#be-the-pr-wrangler-for-a-week).
+- Участвуйте в [графике дежурства смотрителя PR](https://github.com/kubernetes/website/wiki/PR-Wranglers), чтобы вас назначили дежурным проверяющим на неделю. SIG Docs ожидает, что все утверждающие примут участие в этом графике. За подробностям обратитесь к странице [Be the PR Wrangler for a week](/ru/docs/contribute/advanced#дежурный-по-pr-на-неделю).
```
Comment (php-coder) [inline content/ru/docs/contribute/participating.md]:
```suggestion
- Участвуйте в [графике дежурства смотрителя PR](https://github.com/kubernetes/website/wiki/PR-Wranglers), чтобы вас назначили дежурным проверяющим на неделю. SIG Docs ожидает, что все утверждающие примут участие в этом графике. За подробностями обратитесь к странице [Дежурный по PR на неделю](/ru/docs/contribute/advanced#дежурный-по-pr-на-неделю).
```

File: content/ru/docs/contribute/start.md [hunk 4]
```diff
@@ -66,7 +66,7 @@ SIG Docs совместными усилиями вносит изменения
 
 - **Для существующей страницы**
 
-    Если заметили проблему на существующей странице в [документации Kubernetes](/docs/), перейдите в конец страницы и нажмите кнопку **Create an Issue**. Если вы ещё не авторизованы в GitHub, сделайте это. После этого откроется страница с форма для создания нового запроса в GitHub с уже предварительно заполненным полями.
+    Если заметили проблему на существующей странице в [документации Kubernetes](/ru/docs/), перейдите в конец страницы и нажмите кнопку **Create an Issue**. Если вы ещё не авторизованы в GitHub, сделайте это. После этого откроется страница с форма для создания нового запроса в GitHub с уже предварительно заполненным полями.
```
Comment (php-coder) [inline content/ru/docs/contribute/start.md]:
```suggestion
    Если заметили проблему на существующей странице в [документации Kubernetes](/ru/docs/), перейдите в конец страницы и нажмите кнопку **Create an Issue**. Если вы ещё не авторизованы в GitHub, сделайте это. После этого откроется страница с формой для создания нового запроса в GitHub с уже предварительно заполненным полями.
```

File: content/ru/docs/contribute/start.md [hunk 5]
```diff
@@ -116,7 +116,7 @@ SIG Docs совместными усилиями вносит изменения
 
 ### Поиск задач для работы
 
-Если вы уже нашли что исправить, просто следуйте инструкциям ниже. Для этого вам не обязательно [создавать ишью](#file-actionable-issues) (хотя вы, безусловно, пойти этим путём).
+Если вы уже нашли что исправить, просто следуйте инструкциям ниже. Для этого вам не обязательно [создавать ишью](#создание-хороших-заявок) (хотя вы, безусловно, пойти этим путём).
```
Comment (php-coder) [inline content/ru/docs/contribute/start.md]:
```suggestion
Если вы уже нашли что исправить, просто следуйте инструкциям ниже. Для этого вам не обязательно [создавать ишью](#создание-хороших-заявок) (хотя вы, безусловно, можете пойти этим путём).
```

**Comment by php-coder** [comment]
@lex111 Hi,

I'm sorry for being late with feedback. Could you address it in another PR? If you don't have time, please, let me know and I'll prepare PR myself.

Thanks!

**Pull request merged**